Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: clean package package-deps package-source package-upload package-wheel
+
+package-deps:
+	pip install --upgrade twine wheel
+
+package-source:
+	python setup.py sdist
+
+package-wheel: package-deps
+	python setup.py bdist_wheel --universal
```

Comment:
I'm fine keeping it right now for the sake of testing, as long as we've clearly documented the Python 3 support as WIP.